examples/existing_vpc/main.tf

```diff
@@ -0,0 +1,58 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.0"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+#############################################################################
+# Provision VPC
+#############################################################################
+
+resource "ibm_is_vpc" "example_vpc" {
+  name           = "${var.prefix}-vpc"
+  resource_group = module.resource_group.resource_group_id
+  tags           = var.resource_tags
+}
+
+module "slz_vpc" {
+  source            = "../../"
+  resource_group_id = module.resource_group.resource_group_id
+  region            = var.region
+  tags              = var.resource_tags
+  create_vpc        = false
+  existing_vpc_id   = resource.ibm_is_vpc.example_vpc.id
+  create_subnets    = false
+
+  subnets = {
```

Comment:
You have set `create_subnets` to `false` here, so I expect to see `existing_subnet_ids` to be used, so you will also need to create subnets in the "example_vpc" VPC before calling this module.